content/en/docs/reference/resource-types/_index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Resource Types
```

Comment:
I think the title of the page (and the directory) should be changed. I was initially confused
by the use of the "resource".

Reply:
Would "PersistentVolumeClaim Resource Types" clarify it further?